examples/docker/02_dind/on-create.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Start Docker in the background
+sudo -u root /bin/sh -c 'nohup dockerd 2>&1 > /var/log/docker.log &'
```

Comment:
```suggestion
sudo -u root /bin/sh -c 'nohup dockerd > /var/log/docker.log &'
```
Nohup already redirects stderr and the order here was probably wrong, since `2>&1` before `1> file`, stderr would still go to the old stdout.

Reply:
I have stories to tell you about how this can go wrong spectacularly!